symplyphysics/laws/thermodynamics/volume_of_heated_liquid_from_initial_volume_and_heating_temperature.py

```diff
@@ -0,0 +1,42 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+                           validate_output)
+
+# Description
+## As the temperature increases, the intensity of the thermal motion of the particles of matter increases. This leads to the fact that the molecules are more "actively" repelled from each other.
+## As the empty spaces between molecules increase, most bodies increase in size when heated.

```

Comment:
It's not always true. For example, gas can be constrained in some volume and instead the pressure rises. Add conditions. And I suggest to write a proof to be sure we know all conditions.